src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -176,10 +176,66 @@ public void initialize(URL location, ResourceBundle resources) {
         newMessageIndicator.visibleProperty().bind(newMessageIndicator.textProperty().isNotEqualTo("0"));
     }
 
+    @FXML
+    BorderPane quotaBlock;
+    @FXML
+    Label quota;
+    @FXML
+    Label provider;
+    @FXML
+    Label quotaDescription;
+    @Inject
+    BoxClient boxClient;
+
+    private void fillQuotaInformation() {
+        try {
+            quotaState = boxClient.getQuotaState();
+        } catch (IOException | QblInvalidCredentials e) {
+            alert(e);
+        }
+        if (quotaState == null) {
```

Comment:
as far as I can see, this happens only in the catch-case of `boxClient.getQuotaState();`, move it there to clear your intent. The if gets unneccessary then